x-pack/plugins/task_manager/server/plugin.ts

```diff
@@ -267,11 +281,29 @@ export class TaskManagerPlugin
       requestTimeouts: this.config.request_timeouts,
     });
 
+    const defaultCapacity = getDefaultCapacity({
+      claimStrategy: this.config?.claim_strategy,
+      heapSizeLimit: this.heapSizeLimit,
+      isCloud: cloud?.isCloudEnabled ?? false,
+      isServerless: !!serverless,
```

Comment:
nit: you may want to check `this.initContext.env.packageInfo.buildFlavor === 'serverless'` and save yourself from one additional plugin dependency 😇

Reply:
Oh good tip! Thanks!

Reply:
Updated in https://github.com/elastic/kibana/pull/189626/commits/61f587b252cf73838fdb3ccf0c2180e561edc071